native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe/deserializer/IDataFormatDeserializer.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+using AWSGsrSerDe.common;
+
+namespace AWSGsrSerDe.deserializer
+{
+    public interface IDataFormatDeserializer
+    {
+        void Configure(GlueSchemaRegistryConfiguration configs);
```

Comment:
Cache need not be configurable for this as the data won't change by time. We can start with a constant capacity for the cache. Making it configurable can be a TODO. By that way `configure` method can be removed.

Reply:
I just realized that and I removed it.